etc/install_julia_gap.sh

```diff
@@ -5,15 +5,17 @@ set -e
 main_dir=${PWD}
 
 # Download and install Julia
-wget https://julialangnightlies-s3.julialang.org/bin/linux/x64/julia-latest-linux64.tar.gz
-mkdir julia 
-tar xf julia-latest-linux64.tar.gz -C julia --strip-components 1
+if [[ $DONT_FETCH_JULIA != yes ]]; then
+    wget https://julialangnightlies-s3.julialang.org/bin/linux/x64/julia-latest-linux64.tar.gz
+    mkdir julia
+    tar xf julia-latest-linux64.tar.gz -C julia --strip-components 1
```

Comment:
Should we really download a nightly here, and not v1.1? (Of course doesn't have to be in this PR)

Reply:
I will do something completely different. I will delete this script (as it is not super useful anyway anymore) by just the GAP install script and rename it to `travis_install_gap.sh`